website/docs/cloud-docs/workspaces/organize-workspaces-with-projects.mdx

```diff
@@ -0,0 +1,95 @@
+ ---
+page_title: Organizing Workspaces with Projects
+description: |-
+Use projects to organize and group workspaces and create ownership boundaries across your infrastructure.
+—
```

Comment:
Tiny formatting tweak to the page metadata:

```suggestion
---
page_title: Organizing Workspaces with Projects
description: |-
Use projects to organize and group workspaces and create ownership boundaries across your infrastructure.
---
```

Reply:
Changes made in this PR to be merged upon approval: https://github.com/hashicorp/terraform-docs-common/pull/245